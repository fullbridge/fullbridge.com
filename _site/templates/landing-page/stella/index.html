<!doctype html>
<html class="no-js" lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="x-ua-compatible" content="ie=edge">

    <!-- Mobile view styles -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" />
    <meta http-equiv="cleartype" content="on">

    <!-- SEO stuff -->
    
    <meta name="robots" content="noindex, nofollow">
    
    <meta name="author" content="Fullbridge">
    <meta name="description" content="Fullbridge">
    <meta property="og:description" content="Fullbridge" />
    <meta property="og:type" content="website" />
    <meta property="og:image" content="touch-icon-ipad-retina.png" />
    <meta property="og:url" content="http://fullbridge.com/templates/landing-page/stella/" />
    
    <meta property="og:site_name" content="Stella Landing | Fullbridge" />
    <meta property="og:title" content="Stella Landing | Fullbridge" />
    <title>Stella Landing | Fullbridge</title>
    
    <link rel="canonical" href="http://fullbridge.com/templates/landing-page/stella/">

    <!-- Thumbnails -->
    <link rel="image_src" href="/apple-touch-icon.png" />
    <link rel="apple-touch-icon" href="/apple-touch-icon.png">
    <link rel="icon" href="/favicon.ico">

    <!-- CSS -->
    <link rel="stylesheet" href="/css/main.css">
    <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,300italic,700,700italic|Roboto+Condensed:400">
    <link rel="stylesheet" href="https://d1azc1qln24ryf.cloudfront.net/68795/Fullbridgemarketingsite/style-cf.css?t9s27s">

    <!-- JS -->
    <script src="/js/vendor/modernizr-2.8.3.min.js"></script>

    
    <!-- site.ga-id is a variable in the config.yml -->
    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-21169274-2', 'auto');
      ga('send', 'pageview');

    </script>
    
  </head>
  <body  data-current-page="template">

      <nav role="navigation" class="main-menu" id="js-menu-bar">
      <div class="main-menu-wrapper">
        
        <div class="main-menu-logo">
        
          <a href="/" title="Fullbridge">
            <img src="/img/logo-fullbridge.png" alt="Fullbridge">
            <span>Fullbridge</span>
          </a>
        
        </div>
        
        <ul class="main-menu-items">
          <li class="main-menu-item is-dropdown">
            <strong class="trigger">
              Products
            </strong>
            <ul>
              <li>
                <a href="/product/learning-labs">
                  Learning Labs
                </a>
              </li>
              <li>
                <a href="/product/working-labs">
                  Working Labs
                </a>
              </li>
              <!-- <li>
                <a href="/product/learning-modules">
                  Learning Modules
                </a>
              </li> -->
            </ul>
          </li>
          <li class="main-menu-item is-dropdown">
            <strong class="trigger">
              Who we help
            </strong>
            <ul>
              <li>
                <a href="/corporate">
                  Corporate
                </a>
              </li>
              <li>
                <a href="/law">
                  Law
                </a>
              </li>
              <li>
                <a href="/higher-ed">
                  Higher-Ed
                </a>
              </li>
            </ul>
          </li>
          <li class="main-menu-item">
            <a href="/lyra-technology" class="trigger">
              Lyra&trade;
            </a>
          </li>
          <li class="main-menu-item is-full-menu">
            <a href="javascript:" class="trigger">
              <span>View full menu</span>
            </a>
            <div class="full-menu">
  <ul class="full-menu-items">
    <li class="full-menu-item is-primary">
      <ul>
        <li>
          <a href="/team">
            Team
          </a>
        </li>
        <li>
          <a href="/global-partners">
            Global Partners
          </a>
        </li>
        <!-- <li>
          <a href="//blog.fullbridge.com">
            Blog
          </a>
        </li> -->
        <li>
          <a href="//platform.fullbridge.com">
            Log in to Lyra
          </a>
        </li>
      </ul>
    </li>
    <hr role="separator">
    <li class="full-menu-item is-secondary">
      <strong>
        What we do
      </strong>
      <ul>
        <li>
          <a href="/the-future-of-learning">
            The Future of Learning
          </a>
        </li>
        <li>
          <a href="/a-culture-of-learning">
            A Culture of Learning
          </a>
        </li>
        <li>
          <a href="/lyra-technology">
            Lyra&trade; Technology
          </a>
        </li>
      </ul>
    </li>
    <li class="full-menu-item is-secondary">
      <strong>
        Who we help
      </strong>
      <ul>
        <li>
          <a href="/corporate">
            Corporate
          </a>
        </li>
        <li>
          <a href="/law">
            Law
          </a>
        </li>
        <li>
          <a href="/higher-ed">
            Higher-Ed
          </a>
        </li>
      </ul>
    </li>
    <li class="full-menu-item is-secondary">
      <strong>
        Products
      </strong>
      <ul>
        <li>
          <a href="/product/learning-labs">
            Learning Labs
          </a>
        </li>
        <li>
          <a href="/product/working-labs">
            Working Labs
          </a>
        </li>
        <!-- <li>
          <a href="/product/learning-modules">
            Learning Modules
          </a>
        </li> -->
      </ul>
    </li>
    <li class="full-menu-item is-secondary">
      <strong>
        Engage
      </strong>
      <ul>
        <li>
          <a href="#contact-us">
            Contact Us
          </a>
        </li>
        <li>
          <div class="social-icons">
            <a href="//www.linkedin.com/company/fullbridge" class="icon-linkedin" target="_blank">
              <span>LinkedIn</span>
            </a>
            <a href="//www.facebook.com/fullbridgeinc/" class="icon-facebook" target="_blank">
              <span>Facebook</span>
            </a>
            <a href="https://twitter.com/fullbridge_inc" class="icon-twitter" target="_blank">
              <span>Twitter</span>
            </a>
          </div>
        </li>
      </ul>
    </li>
    <li class="full-menu-item is-secondary">
      <strong>Fullbridge, Inc.</strong>
      <ul>
        <li>
          38 Chauncy Street<br>
          Suite 1100<br>
          Boston, MA 02111<br>
          p: +1 (857) 233-2323<br>
          e: programs@fullbridge.com
        </li>
      </ul>
    </li>
  </ul>
</div>

          </li>
          <li class="main-menu-item is-cta">
            <a href="#contact-us">
              <span>Request a</span> demo
            </a>
          </li>
        </ul>
      </div>
      <div class="modal" id="contact-us">
        <a href="#!" class="modal-shadow"></a>
        <div class="modal-content">
          <div class="hubspot-form is-contact">
  <!--[if lte IE 8]>
  <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2-legacy.js"></script>
  <![endif]-->
  <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2.js"></script>
  <script>
   hbspt.forms.create({
     css: "",
     portalId: "162012",
     formId: "d3f1fcd9-46e2-48b8-8ae0-a6db8edb2aa0"
   });
  </script>
</div>

        </div>
      </div>
    </nav>

  <main>
  <header role="banner">
    <div class="landing-stella-hero">
      <div class="landing-wrapper">
        <div class="landing-stella-hero-headlines">
          <h1>
            Headline that matches what was clicked
          </h1>
          <h2>
            A supporting subheader to provide extra info
          </h2>
          <a href="javascript:" class="button is-primary">
            <span>Action</span>
          </a>
        </div>
        <figure class="landing-stella-hero-graphic">
          <img src="http://dummyimage.com/800x600/4d494d/686a82.gif&text=hero+image" alt="hero+image">
        </figure>
      </div>
    </div>
  </header>
  <hr role="separator" class="divider">
  <article role="article">
    <div class="landing-section">
      <div class="landing-wrapper">
        <div class="even-columns has-gutters">
          <div class="even-column">
            <section role="region">
              <header role="heading">
                <h2>Make a point here</h2>
              </header>
              <p>
                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam in dui mauris. Vivamus hendrerit arcu sed erat molestie vehicula. Sed auctor neque eu tellus rhoncus ut eleifend nibh porttitor. Ut in nulla enim. Phasellus molestie magna non est bibendum non venenatis nisl tempor. Suspendisse dictum feugiat nisl ut dapibus. Mauris iaculis porttitor posuere. Praesent id metus massa, ut blandit odio. Proin quis tortor orci. Etiam at risus et justo dignissim congue. Donec congue lacinia dui, a porttitor lectus condimentum laoreet. Nunc eu ullamcorper orci. (max length: 580 characters)
              </p>
            </section>
          </div>
          <div class="even-column">
            <section role="region">
              <header role="heading">
                <h2>Make another point here</h2>
              </header>
              <p>
                Nunc eu ullamcorper orci. Quisque eget odio ac lectus vestibulum faucibus eget in metus. In pellentesque faucibus vestibulum. Nulla at nulla justo, eget luctus tortor. Nulla facilisi. Duis aliquet egestas purus in blandit. Curabitur vulputate, ligula lacinia scelerisque tempor, lacus lacus ornare ante, ac egestas est urna sit amet arcu. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Sed molestie augue sit amet leo consequat posuere. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia. (max length: 580 characters)
              </p>
            </section>
          </div>
        </div>
      </div>
    </div>
    <hr role="separator" class="divider">
    <section role="region">
      <header role="heading" class="landing-dividing-header">
        <div class="landing-wrapper">
          <h2>
            A reinforcement statement to maintain interest
          </h2>
        </div>
      </header>
      <hr role="separator" class="divider">
      <div class="landing-section">
        <div class="landing-wrapper">
          <div class="even-columns has-gutters">
            <div class="even-column">
              <h3>Benefit 1 Title</h3>
              <p>
                This is content describing the first benefit of your product or service. It should act as a supporting description of the benefit statements you wrote above. (ideal length: 200 characters)
              </p>
            </div>
            <div class="even-column">
              <h3>Benefit 2 Title</h3>
              <p>
                This is content describing the first benefit of your product or service. It should act as a supporting description of the benefit statements you wrote above. (ideal length: 200 characters)
              </p>
            </div>
            <div class="even-column">
              <h3>Benefit 3 Title</h3>
              <p>
                This is content describing the first benefit of your product or service. It should act as a supporting description of the benefit statements you wrote above. (ideal length: 200 characters)
              </p>
            </div>
          </div>
          <div class="even-columns has-gutters">
            <div class="even-column">
              <h3>Benefit 4 Title</h3>
              <p>
                This is content describing the first benefit of your product or service. It should act as a supporting description of the benefit statements you wrote above. (ideal length: 200 characters)
              </p>
            </div>
            <div class="even-column">
              <h3>Benefit 5 Title</h3>
              <p>
                This is content describing the first benefit of your product or service. It should act as a supporting description of the benefit statements you wrote above. (ideal length: 200 characters)
              </p>
            </div>
            <div class="even-column">
              <h3>Benefit 6 Title</h3>
              <p>
                This is content describing the first benefit of your product or service. It should act as a supporting description of the benefit statements you wrote above. (ideal length: 200 characters)
              </p>
            </div>
          </div>
        </div>
      </div>
    </section>
    <hr role="separator" class="divider">
    <section role="region">
      <header role="heading" class="landing-dividing-header">
        <div class="landing-wrapper">
          <h2>
            The closing argument statement
          </h2>
        </div>
      </header>
      <hr role="separator" class="divider">
      <div class="landing-wrapper">
        <div class="landing-closing">
          <p class="landing-big-emphasis">
            This is closing content that sums up your offering. It should follow on from your closing argument statement. (ideal length: 110 characters)
          </p>
          <p>
            <a href="#contact-us" class="button is-primary">
              <span>Request a demo</span>
            </a>
          </p>
        </div>
      </div>
    </section>
  </article>
</main>

  <footer class="main-footer">
  <div class="full-menu">
  <ul class="full-menu-items">
    <li class="full-menu-item is-primary">
      <ul>
        <li>
          <a href="/team">
            Team
          </a>
        </li>
        <li>
          <a href="/global-partners">
            Global Partners
          </a>
        </li>
        <!-- <li>
          <a href="//blog.fullbridge.com">
            Blog
          </a>
        </li> -->
        <li>
          <a href="//platform.fullbridge.com">
            Log in to Lyra
          </a>
        </li>
      </ul>
    </li>
    <hr role="separator">
    <li class="full-menu-item is-secondary">
      <strong>
        What we do
      </strong>
      <ul>
        <li>
          <a href="/the-future-of-learning">
            The Future of Learning
          </a>
        </li>
        <li>
          <a href="/a-culture-of-learning">
            A Culture of Learning
          </a>
        </li>
        <li>
          <a href="/lyra-technology">
            Lyra&trade; Technology
          </a>
        </li>
      </ul>
    </li>
    <li class="full-menu-item is-secondary">
      <strong>
        Who we help
      </strong>
      <ul>
        <li>
          <a href="/corporate">
            Corporate
          </a>
        </li>
        <li>
          <a href="/law">
            Law
          </a>
        </li>
        <li>
          <a href="/higher-ed">
            Higher-Ed
          </a>
        </li>
      </ul>
    </li>
    <li class="full-menu-item is-secondary">
      <strong>
        Products
      </strong>
      <ul>
        <li>
          <a href="/product/learning-labs">
            Learning Labs
          </a>
        </li>
        <li>
          <a href="/product/working-labs">
            Working Labs
          </a>
        </li>
        <!-- <li>
          <a href="/product/learning-modules">
            Learning Modules
          </a>
        </li> -->
      </ul>
    </li>
    <li class="full-menu-item is-secondary">
      <strong>
        Engage
      </strong>
      <ul>
        <li>
          <a href="#contact-us">
            Contact Us
          </a>
        </li>
        <li>
          <div class="social-icons">
            <a href="//www.linkedin.com/company/fullbridge" class="icon-linkedin" target="_blank">
              <span>LinkedIn</span>
            </a>
            <a href="//www.facebook.com/fullbridgeinc/" class="icon-facebook" target="_blank">
              <span>Facebook</span>
            </a>
            <a href="https://twitter.com/fullbridge_inc" class="icon-twitter" target="_blank">
              <span>Twitter</span>
            </a>
          </div>
        </li>
      </ul>
    </li>
    <li class="full-menu-item is-secondary">
      <strong>Fullbridge, Inc.</strong>
      <ul>
        <li>
          38 Chauncy Street<br>
          Suite 1100<br>
          Boston, MA 02111<br>
          p: +1 (857) 233-2323<br>
          e: programs@fullbridge.com
        </li>
      </ul>
    </li>
  </ul>
</div>

</footer>

    <script src="/js/main.js"></script>

    

    
    <!-- site.hubspot-id is a variable in the config.yml -->
    <script type="text/javascript" id="hs-script-loader" async defer src="//js.hs-scripts.com/162012.js"></script>
    
  </body>
</html>

