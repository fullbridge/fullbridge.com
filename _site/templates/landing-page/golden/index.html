<!doctype html>
<html class="no-js" lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="x-ua-compatible" content="ie=edge">

    <!-- Mobile view styles -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" />
    <meta http-equiv="cleartype" content="on">

    <!-- SEO stuff -->
    <meta name="author" content="Fullbridge">
    <meta name="description" content="Fullbridge">
    <meta property="og:description" content="Fullbridge" />
    <meta property="og:type" content="website" />
    <meta property="og:image" content="touch-icon-ipad-retina.png" />
    <meta property="og:url" content="http://fullbridge.com/templates/landing-page/golden/" />
    
    <meta property="og:site_name" content="Landing page | Fullbridge" />
    <meta property="og:title" content="Landing page | Fullbridge" />
    <title>Landing page | Fullbridge</title>
    
    <link rel="canonical" href="http://fullbridge.com/templates/landing-page/golden/">

    <!-- Thumbnails -->
    <link rel="image_src" href="/apple-touch-icon.png" />
    <link rel="apple-touch-icon" href="/apple-touch-icon.png">
    <link rel="icon" href="/favicon.ico">

    <!-- CSS -->
    <link rel="stylesheet" href="/css/main.css">
    <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,300italic,700,700italic|Roboto+Condensed:400">

    <!-- JS -->
    <script src="/js/vendor/modernizr-2.8.3.min.js"></script>
  </head>
  <body  data-current-page="">

      <nav role="navigation" class="main-menu">
      <div class="main-menu-wrapper">
        
        <h2 class="main-menu-logo">
        
          <a href="http://fullbridge.com" title="Fullbridge">
            <img src="/img/logo-fullbridge.png" alt="Fullbridge">
            <span>Fullbridge</span>
          </a>
        
        </h2>
        
        <ul class="main-menu-items">
          <li class="main-menu-item is-dropdown">
            <strong class="trigger">
              Products
            </strong>
            <ul>
              <li>
                <a href="javascript:">
                  Learning Labs
                </a>
              </li>
              <li>
                <a href="javascript:">
                  Working Labs
                </a>
              </li>
              <li>
                <a href="javascript:">
                  Learning Modules
                </a>
              </li>
            </ul>
          </li>
          <li class="main-menu-item is-dropdown">
            <strong class="trigger">
              Who we help
            </strong>
            <ul>
              <li>
                <a href="javascript:">
                  Corporate
                </a>
              </li>
              <li>
                <a href="javascript:">
                  Law
                </a>
              </li>
              <li>
                <a href="javascript:">
                  Higher-Ed
                </a>
              </li>
            </ul>
          </li>
          <li class="main-menu-item is-full-menu">
            <a href="javascript:" class="trigger">
              <span>View full menu</span>
            </a>
            <div class="full-menu">
  <ul class="full-menu-items">
    <li class="full-menu-item is-primary">
      <ul>
        <li>
          <a href="javascript:">
            Team
          </a>
        </li>
        <li>
          <a href="javascript:">
            Global Partners
          </a>
        </li>
        <li>
          <a href="javascript:">
            Media
          </a>
        </li>
        <li>
          <a href="javascript:">
            Outreach
          </a>
        </li>
        <li>
          <a href="//platform.fullbridge.com">
            Log in to Lyra
          </a>
        </li>
      </ul>
    </li>
    <hr role="separator">
    <li class="full-menu-item is-secondary">
      <strong>
        What we do
      </strong>
      <ul>
        <li>
          <a href="javascript:">
            The Future of Learning
          </a>
        </li>
        <li>
          <a href="javascript:">
            A Culture of Learning
          </a>
        </li>
        <li>
          <a href="javascript:">
            Lyra&trade; Technology
          </a>
        </li>
      </ul>
    </li>
    <li class="full-menu-item is-secondary">
      <strong>
        Who we help
      </strong>
      <ul>
        <li>
          <a href="/corporate">
            Corporate
          </a>
        </li>
        <li>
          <a href="/law">
            Law
          </a>
        </li>
        <li>
          <a href="higher-ed">
            Higher-Ed
          </a>
        </li>
      </ul>
    </li>
    <li class="full-menu-item is-secondary">
      <strong>
        Products
      </strong>
      <ul>
        <li>
          <a href="javascript:">
            Learning Labs
          </a>
        </li>
        <li>
          <a href="javascript:">
            Working Labs
          </a>
        </li>
        <li>
          <a href="javascript:">
            Learning Modules
          </a>
        </li>
      </ul>
    </li>
    <li class="full-menu-item is-secondary">
      <strong>
        Engage
      </strong>
      <ul>
        <li>
          <a href="#contact-us">
            Contact Us
          </a>
        </li>
        <li>
          <a href="javascript:">
            Careers
          </a>
        </li>
        <li>
          <a href="javascript:">
            Webinars
          </a>
        </li>
      </ul>
    </li>
    <li class="full-menu-item is-secondary">
      <strong>Fullbridge, Inc.</strong>
      <ul>
        <li>
          38 Chauncy Street<br>
          Suite 1100<br>
          Boston, MA 02111<br>
          p: +1 (857) 233-2323<br>
          e: programs@fullbridge.com
        </li>
      </ul>
    </li>
  </ul>
</div>

          </li>
          <li class="main-menu-item is-cta">
            <a href="#contact-us">
              <span>Request a</span> demo
            </a>
          </li>
        </ul>
      </div>
      <div class="modal" id="contact-us">
        <a href="#" class="modal-shadow"></a>
        <div class="modal-content">
          <div class="hubspot-form">
  <!--[if lte IE 8]>
  <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2-legacy.js"></script>
  <![endif]-->
  <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2.js"></script>
  <script>
   hbspt.forms.create({
     css: "",
     portalId: "162012",
     formId: "d3f1fcd9-46e2-48b8-8ae0-a6db8edb2aa0"
   });
  </script>
</div>

        </div>
      </div>
    </nav>

  <main>
  <header role="banner">
    <div class="landing-hero">
      <div class="landing-hero-headlines">
        <div class="landing-wrapper">
          <h1>
            Headline that matches what was clicked
          </h1>
          <h2>
            A supporting subheader to provide extra info
          </h2>
        </div>
      </div>
      <hr class="divider" />
      <div class="landing-wrapper">
        <div class="landing-cta">
          <div class="landing-cta-main">
            <figure>
              <img src="http://dummyimage.com/1000x400/4d494d/686a82.gif&text=hero+image" alt="placeholder+image">
            </figure>
            <p class="landing-big-emphasis">
              <strong>This is introductory content</strong> that explains your offering in more detail. It should follow on from your headline and explain the benefits of your product or service.
            </p>
            <ol class="landing-numbered-list">
              <li>Your first feature written in the form of a benefit statement.</li>
              <li>Your second feature written in the form of a benefit statement.</li>
              <li>Your third feature written in the form of a benefit statement.</li>
            </ol>
          </div>
          <aside class="landing-cta-aside">
            <div class="landing-cta-form">
              <header>
                <h3>
                  Contact Us
                </h3>
              </header>
              <div class="hubspot-form">
  <!--[if lte IE 8]>
  <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2-legacy.js"></script>
  <![endif]-->
  <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2.js"></script>
  <script>
   hbspt.forms.create({
     css: "",
     portalId: "162012",
     formId: "d3f1fcd9-46e2-48b8-8ae0-a6db8edb2aa0"
   });
  </script>
</div>

            </div>
            <div class="landing-blockquote">
              <figure>
                <img src="/img/logo-holycross.png" alt="The College of Holy Cross">
              </figure>
              <blockquote>
                <quote>
                  You can add a customer testimonial for social proof
                </quote>
                <div>
                  <em>
                    &mdash;John Doe, The College of Holy Cross
                  </em>
                </div>
              </blockquote>
              <!-- <p>
              </p> -->
            </div>
          </aside>
        </div>
      </div>
    </div>
  </header>
  <article role="article">
    <hr class="divider" />
    <section role="region">
      <header role="heading" class="landing-dividing-header">
        <div class="landing-wrapper">
          <h2>
            A reinforcement statement to maintain interest
          </h2>
        </div>
      </header>
      <hr class="divider" />
      <div class="landing-section">
        <div class="landing-wrapper">
          <div class="even-columns has-gutters">
            <div class="even-column">
              <figure>
                <img src="http://dummyimage.com/800x600/4d494d/686a82.gif&text=placeholder+image" alt="placeholder+image">
              </figure>
              <h3>Benefit 1 Title</h3>
              <p>
                This is content describing the first benefit of your product or service. It should act as a supporting description of the benefit statements you wrote above.
              </p>
            </div>
            <div class="even-column">
              <figure>
                <img src="http://dummyimage.com/800x600/4d494d/686a82.gif&text=placeholder+image" alt="placeholder+image">
              </figure>
              <h3>Benefit 2 Title</h3>
              <p>
                This is content describing the first benefit of your product or service. It should act as a supporting description of the benefit statements you wrote above.
              </p>
            </div>
            <div class="even-column">
              <figure>
                <img src="http://dummyimage.com/800x600/4d494d/686a82.gif&text=placeholder+image" alt="placeholder+image">
              </figure>
              <h3>Benefit 3 Title</h3>
              <p>
                This is content describing the first benefit of your product or service. It should act as a supporting description of the benefit statements you wrote above.
              </p>
            </div>
          </div>
          <div class="even-columns has-gutters">
            <div class="even-column">
              <figure>
                <img src="http://dummyimage.com/800x600/4d494d/686a82.gif&text=placeholder+image" alt="placeholder+image">
              </figure>
              <h3>Benefit 4 Title</h3>
              <p>
                This is content describing the first benefit of your product or service. It should act as a supporting description of the benefit statements you wrote above.
              </p>
            </div>
            <div class="even-column">
              <figure>
                <img src="http://dummyimage.com/800x600/4d494d/686a82.gif&text=placeholder+image" alt="placeholder+image">
              </figure>
              <h3>Benefit 5 Title</h3>
              <p>
                This is content describing the first benefit of your product or service. It should act as a supporting description of the benefit statements you wrote above.
              </p>
            </div>
        </div>
      </div>
    </section>
    <hr class="divider" />
    <section role="region">
      <header role="heading" class="landing-dividing-header">
        <div class="landing-wrapper">
          <h2>
            The closing argument statement
          </h2>
        </div>
      </header>
      <hr class="divider" />
      <div class="landing-wrapper">
        <div class="landing-closing">
          <p class="landing-big-emphasis">
            This is closing content that sums up your offering. It should follow on from your closing argument statement.
          </p>
          <p>
            <a href="#contact-us" class="button is-primary">
              <span>Request a demo</span>
            </a>
          </p>
        </div>
      </div>
    </section>
  </article>
</main>

  <footer class="main-footer">
  <div class="full-menu">
  <ul class="full-menu-items">
    <li class="full-menu-item is-primary">
      <ul>
        <li>
          <a href="javascript:">
            Team
          </a>
        </li>
        <li>
          <a href="javascript:">
            Global Partners
          </a>
        </li>
        <li>
          <a href="javascript:">
            Media
          </a>
        </li>
        <li>
          <a href="javascript:">
            Outreach
          </a>
        </li>
        <li>
          <a href="//platform.fullbridge.com">
            Log in to Lyra
          </a>
        </li>
      </ul>
    </li>
    <hr role="separator">
    <li class="full-menu-item is-secondary">
      <strong>
        What we do
      </strong>
      <ul>
        <li>
          <a href="javascript:">
            The Future of Learning
          </a>
        </li>
        <li>
          <a href="javascript:">
            A Culture of Learning
          </a>
        </li>
        <li>
          <a href="javascript:">
            Lyra&trade; Technology
          </a>
        </li>
      </ul>
    </li>
    <li class="full-menu-item is-secondary">
      <strong>
        Who we help
      </strong>
      <ul>
        <li>
          <a href="/corporate">
            Corporate
          </a>
        </li>
        <li>
          <a href="/law">
            Law
          </a>
        </li>
        <li>
          <a href="higher-ed">
            Higher-Ed
          </a>
        </li>
      </ul>
    </li>
    <li class="full-menu-item is-secondary">
      <strong>
        Products
      </strong>
      <ul>
        <li>
          <a href="javascript:">
            Learning Labs
          </a>
        </li>
        <li>
          <a href="javascript:">
            Working Labs
          </a>
        </li>
        <li>
          <a href="javascript:">
            Learning Modules
          </a>
        </li>
      </ul>
    </li>
    <li class="full-menu-item is-secondary">
      <strong>
        Engage
      </strong>
      <ul>
        <li>
          <a href="#contact-us">
            Contact Us
          </a>
        </li>
        <li>
          <a href="javascript:">
            Careers
          </a>
        </li>
        <li>
          <a href="javascript:">
            Webinars
          </a>
        </li>
      </ul>
    </li>
    <li class="full-menu-item is-secondary">
      <strong>Fullbridge, Inc.</strong>
      <ul>
        <li>
          38 Chauncy Street<br>
          Suite 1100<br>
          Boston, MA 02111<br>
          p: +1 (857) 233-2323<br>
          e: programs@fullbridge.com
        </li>
      </ul>
    </li>
  </ul>
</div>

</footer>

    <!--
      Extra shit I might need
      <script src="//code.jquery.com/jquery-1.12.0.min.js"></script>
      <script>window.jQuery || document.write('<script src="js/vendor/jquery-1.12.0.min.js"><\/script>')</script>
      <script src="js/plugins.js"></script>
      <script src="/js/main.js"></script>
    -->
    <!-- Intercom -->
    <script>
      window.intercomSettings = {
        app_id: "e6on1lg6"
      };
    </script>
    <script>(function(){var w=window;var ic=w.Intercom;if(typeof ic==="function"){ic('reattach_activator');ic('update',intercomSettings);}else{var d=document;var i=function(){i.c(arguments)};i.q=[];i.c=function(args){i.q.push(args)};w.Intercom=i;function l(){var s=d.createElement('script');s.type='text/javascript';s.async=true;s.src='https://widget.intercom.io/widget/e6on1lg6';var x=d.getElementsByTagName('script')[0];x.parentNode.insertBefore(s,x);}if(w.attachEvent){w.attachEvent('onload',l);}else{w.addEventListener('load',l,false);}}})()</script>
  </body>
</html>

